gtsam.h

```diff
@@ -2895,6 +2897,133 @@ class BinaryMeasurement {
 typedef gtsam::BinaryMeasurement<gtsam::Unit3> BinaryMeasurementUnit3;
 typedef gtsam::BinaryMeasurement<gtsam::Rot3> BinaryMeasurementRot3;
 
+#include <gtsam/sfm/ShonanAveraging.h>
```

Comment:
A lot of stuff got exported, but no corresponding Python tests, so no way to know if they are correct?

Reply:
There is an entire python repo with tests. I will add that in a different PR

Reply:
I did check wrapper is correctly built ;-)